crates/host/src/lib.rs

```diff
@@ -88,40 +90,47 @@ impl<'a> TryFrom<&'a str> for ResourceRef<'a> {
     }
 }
 
-fn extract_server<'a>(resource_ref: &ResourceRef<'a>) -> Option<&'a str> {
-    match resource_ref {
-        ResourceRef::File(_) => None,
-        ResourceRef::Bindle(resource_ref) | ResourceRef::Oci(resource_ref) => {
-            resource_ref.split('/').next()
-        }
-    }
+fn oci_authority(s: &str) -> Option<&str> {
+    let (l, _) = s.split_once('/')?;
+    Some(l)
```

Comment:
This is used for bindle and OCI, should we name it `registry_authority` or `registry_url`?

Reply:
It's now a method on the ref!